extensions/wikia/ARecoveryEngine/js/PageFair/recovery.js

```diff
@@ -29,7 +29,7 @@ define('ext.wikia.aRecoveryEngine.pageFair.recovery', [
 	}
 
 	function isBlocking() {
-		var isBlocking = !!(win.ads && win.ads.runtime.pf && win.ads.runtime.pf.blocking);
+		var isBlocking = !!(win.ads && win.ads.runtime.pf && win.ads.runtime.pf.recovering);
```

Comment:
it's also needed for ad recovery in https://github.com/Wikia/app/blob/f3a47508e9106a93d299be1cbc486d0e22428d07/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L71

the idea of unblocking btf queue is awesome, thank you! :)